TangoRosStreamer/app/src/main/java/eu/intermodalics/tango_ros_streamer/activities/RunningActivity.java

```diff
@@ -384,11 +385,11 @@ private void saveUuidsNamestoHashMap() {
         mTangoServiceClientNode.callGetMapUuidsService();
     }
 
-    private void getTangoPermission(String permissionType) {
+    private void getTangoPermission(String permissionType, int requestCode) {
         Intent intent = new Intent();
         intent.setAction(REQUEST_TANGO_PERMISSION_ACTION);
         intent.putExtra(EXTRA_KEY_PERMISSIONTYPE, permissionType);
-        startActivityForResult(intent, REQUEST_CODE_TANGO_PERMISSION);
+        startActivityForResult(intent, requestCode);
```

Comment:
The current [code](https://github.com/Intermodalics/tango_ros/blob/d2571aff8b6d69baf60e830488ee44a27a0d1354/TangoRosStreamer/app/src/main/java/eu/intermodalics/tango_ros_streamer/activities/RunningActivity.java#L498) only instantiates Java node in case dataset permissions are granted. Is the same required for ADF permissions?